File: CodingNinja.CPP/C++/Array/sumArray.cpp
#include<iostream>

using namespace std;

void sum(int input,int a[])
{
    int sum=0;
    for(int i=0;i<input;i++)
    {
        sum+=a[i];
    }
    cout<<sum;
}

int main()
{
    int n;
    cout<<"Enter the size of the array:";
    cin>>n;
    int a[n];
    cout<<"Enter the elements of the array:";
    for(int i=0;i<n;i++)
    {
        cin>>a[i];
    }
    cout<<"The sum of the elements of the array is:";
    sum(n,a);
    return 0;
}

File: GEEKSFORGEEKS.JAVA/Array/sumArray.java
import java.util.*;
public class sumArray {
    public static int addition(int array[]) 
    {
      int sum=0;
      for(int i=0;i<array.length;i++)
      {
          sum=sum+array[i];
      }
      return sum;  
    }
    public static void main(String args[]) 
    {
    int n;
    try(Scanner sc=new Scanner(System.in))
    {
    System.out.print("Enter the size of the array:");
    n=sc.nextInt();
    int array[]=new int [n];
    System.out.print("Enter the elements of the array:");
    for(int i=0;i<array.length;i++)
    {
        array[i]=sc.nextInt();
    }
    System.out.println("The addition of all the elements in  array is:"+addition(array));
}
}
}
